screens.json

```diff
@@ -614,5 +614,33 @@
 		"w": 1080,
 		"h": 1920,
 		"d": 5.5
+	},
+	{
+		"name": "Samsung Galaxy S6, S6 Edge",
```

Comment:
They are both the same phone in every way, apart from the external design on the screen on the S6 Edge. Their resolutions are the same, so I decided to put them under one entry like the iPhone 4 and 4S entry already shown in the table.